src/app/Http/Controllers/API/PresentationUserController.php

```diff
@@ -39,7 +39,24 @@ public function subscribe(Presentation $presentation, User $user)
      */
     public function unsubscribe(Presentation $presentation, User $user)
     {
-        //
+        // check that the user isn't already unsubscribed to the presentation
+        if (!$presentation->users()->find($user)) {
+            return response()->json([
+                'success' => false,
+                'message' => "Already unsubscribed",
+            ], 409);
+        }
```

Comment:
No need to to check if user is subscribed to the presentation as if it is not the detach will simply not do anything

Reply:
it was mainly to tell the user he's an idiot and he already unsubscribed

Reply:
Ok so you should use the Response class to send the HTTP status so it's more readable you can use it like `Response::HTTP_CONFLICT` or something like that